src/client/components/pages/entities/cb-review.js

```diff
@@ -0,0 +1,70 @@
+import * as bootstrap from 'react-bootstrap';
+const { Col, Row } = bootstrap;
+import React from 'react';
+import { Rating } from "react-simple-star-rating";
+
+
+function ReviewCard({ reviewData }) {
+    const publishedDate = new Date(reviewData.published_on).toDateString();
+    let reviewText = reviewData.text;
+    if (reviewText.length > 75) {
+        reviewText = reviewText.substring(0, 75) + '...';
+    }
+    const reviewLink = "https://critiquebrainz.org/review/" + reviewData.id;
+    return (
+        <div className='border bg-light p-2 mb-2' >
+            <div>
+                <Rating
+                    className="rating-stars"
+                    ratingValue={0}
+                    initialValue={reviewData.rating}
+                    size={15}
+                    stars={5}
+                    readonly
+                    allowHalfIcon
+                    fillColor="#46433A"
+                    allowHover={false}
+                />
+                <small className='float-right'>
+                    Review by: <b>{reviewData.user.display_name}</b> {publishedDate}
+                </small>
+            </div>
+            <div>
+                {reviewText}
+                <a className='float-right' href={reviewLink}>View &gt;</a>
+            </div>
+        </div>
+    )
+}
+
+function EntityReviews({ entity }) {
+    const reviews = entity.reviews.reviews;
```

Comment:
I think this can be improved:
1. if entity.reviews is undefined, or not an object, this line will throw an error
2. I commented in another comment but we can directly pass the reviews array and entity type as props rather than the whole entity object.